packages/builder/src/builderStore/store/backend.js

```diff
@@ -251,6 +257,11 @@ export const getBackendUiStore = () => {
             state.draftTable.primaryDisplay = field.name
           }
 
+          // Set relationship type
+          if (field.type === "link") {
+            state.draftTable.relationshipType = relationshipType
```

Comment:
Noticed this here, don't think you need to store the relationship type in this call, it probably shouldn't be passed into the function/stored at the table level as it appears it is already stored at the field level - I think this might just have been left over from before the transition to it being a field level property.

Reply:
Yes! That makes sense. Cheers!